tests/test_sensor.py

```diff
@@ -439,17 +475,17 @@ def test_silicon_area():
     print('area(0,0) = ',area_image(0,0))
     assert area_image(0,0) == area_image.array.min()
     # Regression test based on values current for GalSim version 1.6.
```

Comment:
Hm.  This change is actually a little worrisome.  I know we changed how the linear interpolation is done between the old and new, but I'm concerned that new version might not be very accurate.  I think Craig verified the old number between the old approximation and a much finer grid with the Poisson solver to significantly better than 0.2% (which is the level of change here for the central pixel).
The maximum value changing by ~8e-5 seems ok.  But I'm not so sure about the (0,0) change of 2e-3.

Reply:
I wonder if the problem is mostly coming at the corners.  The corners of pixel (0,0) move basically directly inward.  So they are pretty accurate from the (0,0) pixel's own coordinates for that point, as the angle of the 45 degree direction is still pointing exactly at the corner.
For the upper right corner, the (1,1) pixel's lower left direction probably still has a good estimate for this point.
But the (0,1) and (1,0) pixels will not be aiming at the right place for their upper left and lower right directions (respectively), so they will hit different points on the curve that aren't actually at the corner point.  Averaging these two with the two accurate ones will bias the corner position outward.
I'm thinking maybe the right procedure is not to average these values, but rather to trust the side that is closer to the center pixel.  So the central pixel gets all of its own values from the original file that Craig verified.  Then the 4 orthogonal pixels use the central pixel's values for the sides they share with it, but their own values for the rest.  And so forth.  Always trusting the values for the pixel with the smaller |i| or |j| as appropriate.  I think this might give more accurate pixel areas.  Especially (0,0) whose area should be literally the same.

Reply:
As a follow up to this, I think if we switch to a method where the 0,0 pixel is truly identical to the old code, we should add corresponding tests for (0,1), (1,0), and (1,1) (i.e. the next 3 most important pixels to get right).  We should at least know how much their areas change between the old and new descriptions.

Reply:
This is now resolved after modifying the code to store extra corner points in the linear boundaries.